en/components/column-chart.md

```diff
@@ -0,0 +1,71 @@
+
+<img class="responsive-img" src="../images/column_chart_title-off.png" srcset="../images/column_chart_title-off@2x.png 2x" />
+
+## State
+
+The Column Chart has two states - Idle and Hover. In the Hover state a tooltip is shown upon hovering over a data point, which is then focused and the other columns are dimmed out.
+
+<img class="responsive-img" src="../images/column_chart_tooltip-off.png" srcset="../images/column_chart_tooltip-off@2x.png 2x" />
+<img class="responsive-img" src="../images/column_chart_tooltip-on.png" srcset="../images/column_chart_tooltip-on@2x.png 2x" />
+
+## Y-Axis
+
+The Y-axis of the column chart comprises of a Title, Labels and an Axis. The Title and the labels can be changed or hidden. To hide the Title just type a Spacebar in its field and to hide a label set its override to ~No Symbol. By changing the Axis' color to transparent you can also hide it if needed. And to hide all of the Y-axis change its state to ~No Symbol.
+
+<img class="responsive-img" src="../images/column_chart_yaxis.png" srcset="../images/column_chart_yaxis@2x.png 2x" />
+
+## Gridlines
+
+The Gridlines of the Column chart can change their color, or if they need to be hidden just change their color to transparent.
+
+<img class="responsive-img" src="../images/column_chart_gridlines.png" srcset="../images/column_chart_gridlines@2x.png 2x" />
+
+## Data Points
+
+The Data Points (the columns) can be hidden by setting individual columns to ~No Symbol in order to change their number. The annotations can be changed or removed by deleting their content and placing a Spacebar instead.
+
+<img class="responsive-img" src="../images/column_chart_columns.png" srcset="../images/column_chart_columns@2x.png 2x" />
+
+
+## Styling
+
+First and foremost, the Column Chart lets you change the color of the series. It is recommended to change it only for another color of the palette we have defined for data visualizations with nuances named series.1, series.2 and so on till series.10. You may also change the colors of the vertical and horizontal axis, the gridlines, as well as the chart area background and border colors. Titles, labels, annotation values etc. are also customizable via the available text styles in the **Indigo.Design System**.
+
+<img class="responsive-img" src="../images/column_chart_styling.png" srcset="../images/column_chart_styling@2x.png 2x" />
+
+## Usage
+
+Use the Column Chart when you want to see the amount of change over a period of time. Always start you numeric Axis at 0, so you don't misrepresent the data. Don't use different colors for the same category. If you use time to represent the change in the category you should always set it on the horizontal axis. Don't use too active color for the Gridlines as it will make the chart hard to read and will take away the focus. If you need a detailed Time-Series analysis  - consider a Line Chart with a Time-Series for this type of data.
```

Comment:
```suggestion
Use the Column Chart when you want to show the amount of change over a period of time. Always start your numeric Axis at 0 to avoid misrepresenting the data. Also, abstain from using different colors for the same category. If referencing time, e.g. dates or months, to represent the change in the category, you should always set it on the horizontal axis. Use subtle colors for the Gridlines to make the chart easier to the eye. If you need a more detailed time series visual, consider using a Line Chart instead.
```